spiffworkflow-backend/src/spiffworkflow_backend/services/process_model_service.py

```diff
@@ -316,5 +322,5 @@ def __scan_spec(
             with open(spec_path, "w") as wf_json:
                 json.dump(self.WF_SCHEMA.dump(spec), wf_json, indent=4)
         if process_group:
-            spec.process_group_id = process_group.id
+            spec.process_group = process_group.id
```

Comment:
is this intended?

Reply:
There was a lint about it when running pyl, believe it looked right at the time - if you don't think it is ok with reverting and noqa'ing it or whatever other option is best.